presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -503,16 +509,16 @@ private void deleteTableColumnStatistics(HiveIdentity identity, String databaseN
     @Override
     public synchronized void updatePartitionStatistics(HiveIdentity identity, String databaseName, String tableName, String partitionName, Function<PartitionStatistics, PartitionStatistics> update)
```

Comment:
In the next step I will make it `updatePartitionStatistics(HiveIdentity identity, Partition partition, Function<PartitionStatistics, PartitionStatistics> update)`